core/main.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+resource "aws_vpc" "devops-vpc" {
+  cidr_block = "10.0.0.0/16"
+}
```

Comment:
I thought we are implementing VPC in https://github.com/nvminhtue/tyson_devops_ic/issues/5 ?

Also, although I did this in my own IC and created all the individual components of a VPC (Subnets, route tables and associations) I think we use a pre-existing module and pass in arguments to do all that for us. Check easyhotel:

https://github.com/nimblehq/easyhotel-infra-aws/blob/7187b356315a2bd845e874d0c4424228981abef7/modules/vpc/main.tf#L49

Reply:
I add VPC as a backbone to create a AWS provider, the detailed implementation will be handled in #5

Reply:
Do we need to create the VPC though? I just thought we could do it separate. Was it to show / demonstrate the terraform plan - what would be created etc?

Reply:
Yes, to show the terraform plan purpose, just a VPC skeleton, we would fill up that later on anyway